src/plugins/system-greeting/index.js

```diff
@@ -0,0 +1,83 @@
+const Periodic = require( "Periodic" );
+const Listener = require( "Listener" );
+
+class SystemGreeting
+{
+    constructor( name, deps )
+    {
+        console.log( "SystemGreeting plugin loaded" );
```

Comment:
Deferring any logging work until the new Pino  PR lands. Will upgrade to use that when it's ready.